spotify_player/src/state/model.rs

```diff
@@ -334,6 +336,12 @@ impl From<rspotify_model::FullAlbum> for Album {
             id: album.id,
             release_date: album.release_date,
             artists: from_simplified_artists_to_artists(album.artists),
+            album_type: match album.album_type {
+                AlbumType::Album => "album".to_string(),
+                AlbumType::Single => "single".to_string(),
+                AlbumType::Compilation => "compilation".to_string(),
+                AlbumType::AppearsOn => "appears on".to_string(),
+            },
```

Comment:
let's keep the original representation `Option<AlbumType>` in case we want to process the type in the future. You can implement `fn type(&self) -> String` method for `Album` that returns the type as string

Reply:
Done